wbooks/src/app/screens/dashboard/screens/Detail/components/SuggestionList/index.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
 
+import { BOOK } from '../../../../../../../constants/routes';
+import suggestionActions from '../../../../../../../redux/suggestionAction/actions';
 import Book from '../../../Home/components/BookList/components/Book';
 import './styles.css';
 
-const SuggestionList = () => {
-  return (
-    <div className="suggestion-list">
-      <div className="suggestion-title">
-        <h4>Suggestions</h4>
+class SuggestionList extends React.Component {
+  componentWillMount() {
+    this.props.dispatch(suggestionActions.getSuggestionList());
+  }
```

Comment:
Notice that, in Smart-Dumb pattern, this piece of code should belong to the Smart component, and the render logic to the Dumb component.